atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepQueueUtils.java

```diff
@@ -46,6 +46,9 @@ public final class SweepQueueUtils {
     public static final long RESET_TIMESTAMP = 0L;
     public static final ColumnRangeSelection ALL_COLUMNS = allPossibleColumns();
     public static final int MINIMUM_WRITE_INDEX = -TargetedSweepMetadata.MAX_DEDICATED_ROWS;
+    public static final int DUMMY_SHARD_FOR_NON_SWEEPABLE = 2050;
```

Comment:
comment the bit representation of 2050?

Reply:
Check if we can instead jank the strategy part. It's a janky schema change, but it should work